symfony/client/src/orangehrmAuthenticationPlugin/pages/login.scss

```diff
@@ -48,3 +48,13 @@
     text-decoration: none;
   }
 }
+.orangehrm-login-forgot {
+  display: flex;
+  justify-content: center;
+  margin-top: 1rem;
+  font-size: 14px;
```

Comment:
hard coded font size??

Reply:
not found a scss variable set as 14px

Reply:
rem and oxd-text built in types also can be used.

Reply:
then shall I use **font-size as 14px** 
it's used in **oxd-button--medium** class